crates/rover-client/src/operations/readme/fetch/runner.rs

```diff
@@ -0,0 +1,97 @@
+use crate::blocking::StudioClient;
+use crate::operations::readme::fetch::ReadmeFetchInput;
+use crate::shared::GraphRef;
+use crate::RoverClientError;
+use graphql_client::*;
+
+#[derive(GraphQLQuery)]
+// The paths are relative to the directory where your `Cargo.toml` is located.
+// Both json and the GraphQL schema language are supported as sources for the schema
+#[graphql(
+    query_path = "src/operations/readme/fetch/fetch_query.graphql",
+    schema_path = ".schema/schema.graphql",
+    response_derives = "PartialEq, Debug, Serialize, Deserialize",
+    deprecated = "warn"
+)]
+pub struct ReadmeFetchQuery;
+
+pub fn run(input: ReadmeFetchInput, client: &StudioClient) -> Result<String, RoverClientError> {
+    let graph_ref = input.graph_ref.clone();
+    let data = client.post::<ReadmeFetchQuery>(input.into())?;
+    build_response(data, graph_ref)
+}
+
+fn build_response(
+    data: readme_fetch_query::ResponseData,
+    graph_ref: GraphRef,
+) -> Result<String, RoverClientError> {
+    let graph = data.graph.ok_or(RoverClientError::GraphNotFound {
+        graph_ref: graph_ref.clone(),
+    })?;
+    let variant = graph
+        .variant
+        .ok_or(RoverClientError::GraphNotFound { graph_ref })?;
+    let readme = variant.readme;
+    match readme {
+        Some(v) => Ok(v.content),
+        None => Ok("No README defined".to_string()),
```

Comment:
I think `rover readme fetch` should likely return the "default" README in the case that one isn't set for that variant. If I go to the landing page for the `rover-journey` graph in Studio (https://studio.apollographql.com/graph/rover-journey/home?variant=current) I see the following readme content:


```markdown

# Getting started

Welcome to the **{{ graph.name }}** API! 🎉 Get familiar with available objects in the [Schema Reference]({{ graph.url.reference }}), or try querying this graph using [Explorer]({{ graph.url.explorer }}).

This is the default README content. You can customize it however you like. Graph Maintainers, you can edit to include any information that is relevant for developers working with this graph by clicking the pencil icon in the top right corner. ☝

## What this graph is all about

Describe the purpose and use cases for your graph here. This is where you can tell the story of your API, and all of its deep magic...🦄🌌✨

## Accessing the graph

🛰 You can send operations to this graph at `{{ graph.url.endpoint }}`

📇 The Apollo Registry holds the canonical location of your schema. In the registry, this graph is referred to by its “graph ref”, which is: **{{ graph.ref }}**.

*(Note: you can [download Rover](https://www.apollographql.com/docs/rover/getting-started/), the Apollo CLI tool for working with your schema locally.)*

## How to authenticate to this graph

Authentication information has not been added to this README yet. Be sure to let your API developers know how to authenticate, and where to find the auth tokens they’ll need to authenticate with.🔐 👀

## Running operations

Include some operations in your README to help your API developers get started with your graph. Using a code block designated as  `gql` will render the operation with the Run in Explorer mini-button, and will load the operation in Explorer when clicked!

`gql
query Query {
    id
}
`

### Getting help with this graph

For support working with this graph, contact the Graph Admin via [email(replace me)](mailto:email@email.com) or [chat(put your chat link here)](#chatlink).

```

when i run `rover readme fetch rover-journey`, I get the following output: 

```
Fetching graph variant README of rover-journey@current using credentials from the default profile.
Readme: 

No README defined
```

Reply:
looking at the network requests in studio for this, it doesn't look like the front end is sourcing this "default README" from the API (which... I think it probably should?) my guess is that they have something hard coded like "if readme == null, display default readme" - since we'll want the same experience in rover and studio, this "default" should replace the null response the api gives us

Reply:
Yup that's what studio FE does to display the default readme. You raise a good point that it should probably grab the default from the API though - if we add the same logic here, there's danger of the default in rover and the default in studio getting out of sync 🤔 let me try adding the default to the API side.

Reply:
ok, I've changed the API to return the default readme when readme is null (https://github.com/mdg-private/monorepo/pull/8882) and updated this PR to expect that!

Reply:
testing